<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>page022</class>
 <widget class="QWidget" name="page022">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>272</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>272</width>
    <height>272</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>1280</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">page022</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#page022
{
background-color: rgb(0, 0, 0);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="HeaderPanel" name="headerPanel" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>480</width>
       <height>72</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>90</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_top">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="margin">
      <number>0</number>
     </property>
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="0" column="0" rowspan="3">
      <widget class="ATCMbutton" name="atcmButton_prev">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>DejaVu Sans</family>
         <pointsize>24</pointsize>
        </font>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="whatsThis">
        <string>ATCMbutton widget</string>
       </property>
       <property name="pageName">
        <string notr="true">page021</string>
       </property>
       <property name="remember">
        <bool>false</bool>
       </property>
       <property name="statusvar">
        <string notr="true"/>
       </property>
       <property name="viewStatus">
        <bool>false</bool>
       </property>
       <property name="visibilityVar">
        <string notr="true"/>
       </property>
       <property name="bgColor">
        <color>
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </property>
       <property name="borderColor">
        <color>
         <red>255</red>
         <green>85</green>
         <blue>0</blue>
        </color>
       </property>
       <property name="fontColor">
        <color>
         <red>255</red>
         <green>85</green>
         <blue>0</blue>
        </color>
       </property>
       <property name="bgPressColor">
        <color>
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </property>
       <property name="borderPressColor">
        <color>
         <red>255</red>
         <green>85</green>
         <blue>0</blue>
        </color>
       </property>
       <property name="fontPressColor">
        <color>
         <red>255</red>
         <green>85</green>
         <blue>0</blue>
        </color>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pressText">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="systemicons.qrc">
         <normaloff>:/systemicons/Previous.png</normaloff>:/systemicons/Previous.png</iconset>
       </property>
       <property name="borderWidth">
        <number>0</number>
       </property>
       <property name="borderRadius">
        <number>0</number>
       </property>
       <property name="apparence">
        <enum>QFrame::Plain</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_Stopwatch">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>10</number>
       </property>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="ATCMbutton" name="atcmButton_StartPause">
         <property name="minimumSize">
          <size>
           <width>72</width>
           <height>72</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>DejaVu Sans</family>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="iconSize">
          <size>
           <width>64</width>
           <height>60</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string/>
         </property>
         <property name="whatsThis">
          <string>ATCMbutton widget</string>
         </property>
         <property name="pageName">
          <string notr="true"/>
         </property>
         <property name="statusvar">
          <string notr="true"/>
         </property>
         <property name="viewStatus">
          <bool>false</bool>
         </property>
         <property name="visibilityVar">
          <string notr="true"/>
         </property>
         <property name="bgColor">
          <color>
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </property>
         <property name="borderColor">
          <color>
           <red>0</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </property>
         <property name="bgPressColor">
          <color>
           <red>0</red>
           <green>170</green>
           <blue>170</blue>
          </color>
         </property>
         <property name="borderPressColor">
          <color>
           <red>0</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pressText">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/icons/icons/Start.png</normaloff>:/icons/icons/Start.png</iconset>
         </property>
         <property name="pressIcon">
          <iconset resource="resources.qrc">
           <normaloff>:/icons/icons/Pause.png</normaloff>:/icons/icons/Pause.png</iconset>
         </property>
         <property name="borderWidth">
          <number>3</number>
         </property>
         <property name="borderRadius">
          <number>72</number>
         </property>
         <property name="apparence">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_timer">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>DejaVu Sans</family>
           <pointsize>28</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 255, 255);</string>
         </property>
         <property name="text">
          <string>hh:mm:ss</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="ATCMbutton" name="atcmButton_Reset">
         <property name="minimumSize">
          <size>
           <width>72</width>
           <height>72</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>DejaVu Sans</family>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="iconSize">
          <size>
           <width>64</width>
           <height>64</height>
          </size>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string/>
         </property>
         <property name="whatsThis">
          <string>ATCMbutton widget</string>
         </property>
         <property name="pageName">
          <string notr="true"/>
         </property>
         <property name="statusvar">
          <string notr="true"/>
         </property>
         <property name="viewStatus">
          <bool>false</bool>
         </property>
         <property name="visibilityVar">
          <string notr="true"/>
         </property>
         <property name="bgColor">
          <color>
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </property>
         <property name="borderColor">
          <color>
           <red>0</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </property>
         <property name="bgPressColor">
          <color>
           <red>0</red>
           <green>170</green>
           <blue>170</blue>
          </color>
         </property>
         <property name="borderPressColor">
          <color>
           <red>0</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pressText">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/icons/icons/SkipBackward.png</normaloff>:/icons/icons/SkipBackward.png</iconset>
         </property>
         <property name="borderWidth">
          <number>3</number>
         </property>
         <property name="borderRadius">
          <number>72</number>
         </property>
         <property name="apparence">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="0" column="2" rowspan="3">
      <widget class="ATCMbutton" name="atcmButton_next">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>DejaVu Sans</family>
         <pointsize>24</pointsize>
        </font>
       </property>
       <property name="iconSize">
        <size>
         <width>64</width>
         <height>60</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="whatsThis">
        <string>ATCMbutton widget</string>
       </property>
       <property name="pageName">
        <string notr="true"/>
       </property>
       <property name="remember">
        <bool>false</bool>
       </property>
       <property name="statusvar">
        <string notr="true"/>
       </property>
       <property name="viewStatus">
        <bool>false</bool>
       </property>
       <property name="visibilityVar">
        <string notr="true"/>
       </property>
       <property name="bgColor">
        <color>
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </property>
       <property name="borderColor">
        <color>
         <red>255</red>
         <green>85</green>
         <blue>0</blue>
        </color>
       </property>
       <property name="fontColor">
        <color>
         <red>255</red>
         <green>85</green>
         <blue>0</blue>
        </color>
       </property>
       <property name="bgPressColor">
        <color>
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </property>
       <property name="borderPressColor">
        <color>
         <red>255</red>
         <green>85</green>
         <blue>0</blue>
        </color>
       </property>
       <property name="fontPressColor">
        <color>
         <red>255</red>
         <green>85</green>
         <blue>0</blue>
        </color>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pressText">
        <string/>
       </property>
       <property name="borderWidth">
        <number>0</number>
       </property>
       <property name="borderRadius">
        <number>0</number>
       </property>
       <property name="apparence">
        <enum>QFrame::Plain</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_bottom">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ATCMbutton</class>
   <extends>QPushButton</extends>
   <header>atcmplugin.h</header>
  </customwidget>
  <customwidget>
   <class>HeaderPanel</class>
   <extends>QWidget</extends>
   <header>headerpanel.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="systemicons.qrc"/>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>atcmButton_prev</sender>
   <signal>newPage(const char*,bool)</signal>
   <receiver>page022</receiver>
   <slot>goto_page(const char*,bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>goto_page(const char*,bool)</slot>
 </slots>
</ui>
